### PR TITLE
Improve Progress bar doc

### PR DESCRIPTION
**Pull request by saurabh3949**
@mli @zackchase @madjam @nswamy 

- Improved progress bar doc. 
- Progress bar now outputs to logger instead of stdout.

### CODE REVIEW

File: python/mxnet/callback.py [hunk 1]
```diff
@@ -134,26 +134,35 @@ def __call__(self, param):
 
 
 class ProgressBar(object):
-    """Show a progress bar.
+    """Displays the progress of no. of batches processed
```
Comment (zackchase) [inline python/mxnet/callback.py]: Please change to :

"Displays a progress bar, indicating the percentage of batches processed within each epoch."

File: python/mxnet/callback.py [hunk 2]
```diff
@@ -134,26 +134,35 @@ def __call__(self, param):
...
+    [================] 100.0%
     """
     def __init__(self, total, length=80):
         self.bar_len = length
         self.total = total
 
     def __call__(self, param):
-        """Callback to Show progress bar."""
+        """Callback to show progress bar. Will be called by
+           batch_end_callback internally"""
```
Comment (zackchase) [inline python/mxnet/callback.py]: Please end this sentence in  period:

"... batch_end_callback internally."

**Comment by saurabh3949** [comment]
Thanks @zackchase . I have made the required changes.

File: python/mxnet/callback.py [hunk 3]
```diff
@@ -134,26 +133,34 @@ def __call__(self, param):
 
 
 class ProgressBar(object):
-    """Show a progress bar.
+    """Displays a progress bar, indicating the percentage of batches processed within each epoch.
 
     Parameters
     ----------
     total: int
         total batch size
```
Comment (piiswrong) [inline python/mxnet/callback.py]: shouldn't this be total number of batches?

File: python/mxnet/callback.py [hunk 4]
```diff
@@ -134,26 +133,34 @@ def __call__(self, param):
...
     length: int
```
Comment (piiswrong) [inline python/mxnet/callback.py]: what does length mean?

**Comment by Roshrini** [comment]
Hi @saurabh3949 Thanks for the contribution. Can you please address the comments so that we can merge this?

**Pull request closed without merging**